python/RobotControl/RobotSocketMessages.py

```diff
@@ -13,8 +13,6 @@
 class RobotSocketMessageTypes(Enum):
     Command_finished = auto()
```

Comment:
All the functionality to actually send CommandFinished has been remove, so why not CommandFinished itself? I also no longer understand how it works, so i dont think we would use it? 
I suspect the dashboard connection could provide us with information about when the user script is finished, just as easily?

Reply:
Added to task in new agile job